torchrl/data/replay_buffers/storages.py

```diff
@@ -46,7 +49,8 @@ def __len__(self):
 
 
 class ListStorage(Storage):
-    def __init__(self):
+    def __init__(self, size: int):
```

Comment:
I am not sure if it is a good idea to have `size` as a parameter for this class as it does not correspond to say the max size or the current size.

Reply:
`Writer` should take the `size` here under consideration when determining whether to add more items to the end of the storage. (see: https://github.com/facebookresearch/rl/pull/330#discussion_r940102113)

Reply:
Could we name it max_size perhaps? Would that be if for you @shagunsodhani ?

Reply:
yeah I think `max_size` makes sense, esp given that we want to enforce the size now (https://github.com/facebookresearch/rl/pull/330#discussion_r941767100)

Reply:
Perfect, I changed it to `max_size` now.